### PR TITLE
fix: error when opening atest-desktop on Windows

### PR DESCRIPTION
**Pull request by DWJ-Squirtle**
**What type of PR is this?**

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Review by LinuxSuRen — COMMENTED** [review]
image

It looks like you didn't use your GitHub-related email to commit. You can use `git reset --soft origin/master` to reset the commit history, than push it again via `git push -f`.

File: console/atest-desktop/main.js [hunk 1]
```diff
@@ -108,12 +110,25 @@ app.whenReady().then(() => {
...
+	  log.error('Error writing atest.exe to %s: %s', atestFromHome, e.message);
     }
-  }
+	if (fs.existsSync(atestFromHome)) {
+		log.info('atest.exe file exists at ${atestFromHome}')
+	} else {
+		log.error('Failed to write atest.exe to ${atestFromHome}')
+	}
+	}
+
```
Comment (LinuxSuRen) [inline console/atest-desktop/main.js:131]: Forget to format the codes?

Comment (DWJ-Squirtle) [reply console/atest-desktop/main.js:131]: I'm sorry, this was my test and I forgot to delete it.

Comment (LinuxSuRen) [reply console/atest-desktop/main.js:131]: By the way, please don't forget to sign CLA.

image

**Comment by LinuxSuRen** [comment]
You can squash all commits into one. Or, all commits should have the correct email.

image

File: console/atest-desktop/main.js [hunk 2]
```diff
@@ -15,7 +15,9 @@ log.initialize();
 log.transports.file.level = 'info';
 log.transports.file.resolvePathFn = () => path.join(atestHome, 'log.log');
 
-app.dock.setIcon(path.join(__dirname, "api-testing.png"))
+if (process.platform === 'darwin'){
+		app.dock.setIcon(path.join(__dirname, "api-testing.png"))
```
Comment (LinuxSuRen) [inline console/atest-desktop/main.js:19]: One tab is enough.

File: console/atest-desktop/main.js [hunk 3]
```diff
@@ -108,12 +110,19 @@ app.whenReady().then(() => {
...
+	}else{
+		fs.writeFileSync(atestFromHome, data);
+	}
+	} 
     catch (e) { 
       log.error('Error Code: %s', e.code); 
     }
-  }
+	}
+
```
Comment (LinuxSuRen) [inline console/atest-desktop/main.js:125]: Please keep it well-formatted.

**Review by LinuxSuRen — CHANGES_REQUESTED** [review]

**Pull request closed without merging**